### PR TITLE
Prettier done

### PR DESCRIPTION
**Pull request by mirlek**

### CODE REVIEW

**Comment by mirlek** [comment]
Unfortunately, the Update function is inactive.

File: src/App.js [hunk 1]
```diff
@@ -1,36 +1,188 @@
...
-              {/* <Route path='/read' element={<Read />} /> */}
-              <Route path='/update' element={<Update />} />
-              </Routes>
-                {background && (
-              <Routes>
-                <Route path="modal" element={<Create />} />
-              </Routes>
-      )}
-          </div>
+    <div className="App">
```
Comment (Nikneoberg) [inline src/App.js:6]: Well, the App page is quite large. I propose to divide it into components.

I suggest the following structure:
PageLayout is base component. There are title of page and children items.
- TodoLIst is a component that displays a to-do list.
-- Todo, there will be the creation date, title, description and buttons
--- Button a component that accepts an icon and a click event

File: src/App.js [hunk 2]
```diff
@@ -1,36 +1,188 @@
...
+      <div className="todo-menu">
+        <p>To-DO List</p>
+        <div>
+          <Button className="button-add-tasks" onClick={handleShow}>
+            Add New Task
+            <FontAwesomeIcon icon="fa-solid fa-plus" className="plus-solid" />
+          </Button>
+
+          <Modal show={show} onHide={handleClose}>
```
Comment (Nikneoberg) [inline src/App.js]: I think, Modal is not good for this place

File: src/App.js [hunk 3]
```diff
@@ -1,36 +1,188 @@
...
+            <Modal.Header closeButton>
```
Comment (Nikneoberg) [inline src/App.js]: Please move this part to a new component. We'll work on this at another time.

**Review by Nikneoberg — COMMENTED** [review]
Please have a look on my comments

File: src/App.js (hunk 3 above)
Comment (mirlek) [reply src/App.js]: Done, I moved Modal to the ModalForm component.

File: src/App.js (hunk 1 above)
Comment (mirlek) [reply src/App.js:6]: Done! I changed the structure and split the App.js page into the components as you wrote.

**Review by Nikneoberg — APPROVED** [review]
LGTM

**Pull request merged**